readme.md

```diff
@@ -139,3 +141,5 @@ Finally, open a Pull Request.
 [luis-von-ahn]: http://www.cs.cmu.edu/~biglou/resources/
 
 [racial-slurs]: https://en.wikipedia.org/wiki/List_of_ethnic_slurs
+
+[it-source-wiki]: https://en.wikipedia.org/wiki/Italian_profanity
```

Comment:
```suggestion
[it-source-wiki]: https://en.wikipedia.org/wiki/Italian_profanity

```

Reply:
Fixed both files.

Thanks and yes, I'd like to keep on improving the list since the Wikipedia page lists only the more common Italian insults and vulgarities but the list can be improved a lot.
Before doing an all-encompassing list, including dialects (very relevant in Italian language), I wanted to be sure that you share this kind of inclusive setting (conjugations, declinations, blasphemous words, etc).


I hope it will be useful.

Reply:
All right! Yes I would appreciate that. Well please add commits here and let me know if you need help or when you’re ready!